gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ownership/acceptOwnership.ts

```diff
@@ -44,15 +48,20 @@ export const makeAcceptOwnershipCommand = (contractId: CONTRACT_LIST): SolanaCon
       return [tx]
     }
 
+    readOwner = async () => {
+      const readOwnershipCommand = new this.ReadOwnership(this.flags, this.args)
```

Comment:
The `ReadOwnership` functionality is so simple that I don't think is worth to execute an entire command for it. Let's just query the owner directly